mmengine/runner/runner.py

```diff
@@ -1979,12 +1979,12 @@ def resume(self,
                     'Number of GPU used for current experiment is not '
                     'consistent with resuming from checkpoint')
                 if (self.auto_scale_lr is None
-                        or not self.auto_scale_lr.get('enable', False)):
+                        or self.auto_scale_lr.get('enable', False)):
```

Comment:
Hi @alexander-soare , I think we should keep the `not` and modify the error message from "set `enable` in `auto_scale_lr` to False" to "set `enable` in `auto_scale_lr` to True".

Reply:
Sure, but then I think the rest of the error message needs more clarity. It begins with "Cannot automatically rescale lr in resuming." That sentence alone implies that `auto_scale_lr` should be `False`. And apologies: I'm not aware of the underlying logic as I'm new to MMDet, so I can't suggest a good error message myself.

Reply:
I will try to propose an error message.

Reply:
Nice. I think that's better. One nit though: the use of the language "If you want" suggests this is optional. But the RuntimeError leaves the user with no choice. I would suggest you either change the word choice (see my suggestion below) or you make this a warning rather than an exception.

Reply:
👍I think your refinement makes it better and I will merge this PR.

Reply:
@zhouzaida thanks for the clarification!